### PR TITLE
Directional and bidirectional connection views

### PR DESCRIPTION
**Pull request by justinvdm**

### CODE REVIEW

**Comment by justinvdm** [comment]
I've decided to leave bidirectional connections for another PR, since I don't think I need them to do the routing screen basics.

File: go/base/static/js/src/components/plumbing/connections.js [hunk 1]
```diff
@@ -93,9 +97,40 @@
...
+      id: 'head-arrow'
+    }],
+
+    midArrow: [
+      'Arrow', {
+      width: 12,
+      height: 12,
+      location: 0.5,
+      id: 'mid-arrow'
+    }]
```
Comment (hodgestar) [inline go/base/static/js/src/components/plumbing/connections.js:119]: Is `midArrow` intentionally unused? Seems like a cool idea to have it.

Comment (justinvdm) [inline go/base/static/js/src/components/plumbing/connections.js:119]: Ya, its intentional. I've got a feeling we will be needing it for the dialogue views, so I thought I may as well keep it.

**Comment by hodgestar** [comment]
:+1:

**Pull request merged**